### ISSUE TITLE
Using pretrained Word2vec for sentiment dataset

### ISSUE THREAD

**Issue by serenaroseaini** [huseinzol05/malaya#114]
Salam En. Husein.

Im having problem to use the pre-trained word2vec for the sentiment corpus in your malaya project.
How can i fill the pretrained vectors to the word index from the dataset of size 3600.

Label        text
negative   lebih-lebih lagi dengan kemudahan internet dan laman sosial
positive    boleh memberi teguran kepada parti
negative   adalah memningungkan mengapa masyarakat cina bersorak gembira apabila
positive     kami menurunkan defisit daripada 6.7 peratus pada tahun 2009

embedding_matrix = np.zeros((token_count, EMBEDDING_DIM))
for word, i in word_index.items():
     embedding_vector = word2vec.get(word)  _**#error here**_i can't apply word_vector_news
    if embedding_vector is not None:
      embedding_matrix[i,:] = embedding_vector

The target is to get the weights=[embedding_matrix] for the model that i build

embedding_layer = Embedding(
  token_count,
  EMBEDDING_DIM,
  weights=[embedding_matrix]
  input_length=MAX_SEQUENCE_LENGTH,
  trainable=False)

Please help! i'm tryin to learn text analysis, and quite new. Hoping you help. tq

**Comment by huseinzol05** [comment]
Lebih baik encik menggunakan https://malaya.readthedocs.io/en/latest/load-sentiment.html#Sentence-level sebab outputnya `(EMBEDDING_DIM)` dari pretrained word2vec sebab nanti encik kena aggregate `(token_count, EMBEDDING_DIM)` untuk jadi `(EMBEDDING_DIM)`

**Comment by serenaroseaini** [comment]
hmm, tak berapa faham ni.. hmm,  maksudnye kene aggregate (token_count, EMBEDDING_DIM) untuk jadi embedding_matrix ke embedding_dim? saya nak cari mcm mana nak dapatkan embedding_matrix, x boleh nak map pretrained vector kepada word_index saya

Suggestion en husein tu ialah existing model untuk predict sentiment ya..?

**Comment by huseinzol05** [comment]
Join discord untuk diskus lebih lanjut, https://discord.gg/aNzbnRqt3A
